scripts/closure-compiler/build-devapp-bundle.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+# Script that bundles the dev-app using the Google Closure compiler.
+# This is script is used to verify closure-compatiblity of all Material components.
```

Comment:
it only does the first bits of closure verification of course, we don't know if the property renaming did the right thing. can we also run some of the integration tests on the result (or does this PR do that?)